homeassistant/components/sensor/envisalink.py

```diff
@@ -0,0 +1,68 @@
+"""
+Support for Envisalink sensors (shows panel info).
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.envisalink/
+"""
+import logging
+
+from homeassistant.components.envisalink import (EVL_CONTROLLER,
+                                                 EnvisalinkDevice,
+                                                 SIGNAL_KEYPAD_UPDATE)
+from homeassistant.util import convert
+
+REQUIREMENTS = ['pydispatcher==2.0.5']
```

Comment:
No need for this as it will be installed when dependency `envisalink` gets loaded.